--- particleman.orig/man/ion3.in
+++ particleman/man/ion3.in
@@ -150,8 +150,8 @@
 .SH SEE ALSO
 The Ion home page, \fIhttp://iki.fi/tuomov/ion/\fP
 .PP
-The document "Configuring and extending Ion3 with Lua" found on the
-Ion home page.
+The document "Configuring and extending Ion3 with Lua" found in the
+ion3-doc package.
 .PP
 .I DOCDIR/
 .PP
--- particleman.orig/man/welcome.head
+++ particleman/man/welcome.head
@@ -14,11 +14,10 @@
 accessed with <F12>.
 
 When you feel ready to write customized configuration files (you're going
-to do that in any case), please see the configuration manual available from
-the Ion webpage listed at the end of the user manual below. It may, however,
-be beneficial to become well acquainted with Ion before delving into this 
-so that you have grasped the basic concepts necessary to understand the
-binding configuration files.
+to do that in any case), please see the configuration manual available in
+the ion3-doc package. It may, however, be beneficial to become well
+acquainted with Ion before delving into this so that you have grasped the
+basic concepts necessary to understand the binding configuration files.
 
 
 ---- Manual page follows ----
